### PR TITLE
#4944 Issue: added borders and hover to documents page

### PR DESCRIPTION
**Pull request by khalid827**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4967"></a>

### CODE REVIEW

**Review by devvsakib — APPROVED** [review]
LGTM

**Comment by ManishBisht777** [comment]
you can add a before and after screenshot as well. it will help others reviewing the pr

File: pages/docs/index.js [hunk 1]
```diff
@@ -279,6 +279,21 @@ export default function DocsIndex() {
           content="Discover more people in your LinkFree community"
         />
         <link rel="icon" href="/favicon.ico" />
+        <style>
+          {`
```
Comment (ManishBisht777) [inline pages/docs/index.js:283]: the project uses tailwind can't it be done by using tailwind classes itself

**Review by ManishBisht777 — COMMENTED** [review]
left some comment

File: pages/docs/index.js (hunk 1 above)
Comment (khalid827) [reply pages/docs/index.js:283]: > the project uses tailwind can't it be done by using tailwind classes itself

I will check tomorrow and try to work on it

**Comment by khalid827** [comment]
> you can add a before and after screenshot as well. it will help others reviewing the pr

Thank u for the feedback. I am actually new to contribution and this is my first I will try to do that from next time

**Comment by eddiejaoude** [comment]
Thank you for your contribution 👍 you got assigned the issue but then you removed your assignment to the issue and the maintainers reassign the issue to someone else. I will there fore close your pull request as the issue assignee raised a pull request https://github.com/EddieHubCommunity/LinkFree/pull/5125

**Pull request closed without merging**